dotnet/src/AutoGen.Ollama/AutoGen.Ollama.csproj

```diff
@@ -20,4 +20,8 @@
       <ProjectReference Include="..\AutoGen.Core\AutoGen.Core.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageReference Include="System.Formats.Asn1" />
```

Comment:
Is it still necessary

Reply:
Unsure

Reply:
I just check on my end, it's not necessary and we can revert this change